requirements.txt

```diff
@@ -9,4 +9,5 @@ pytest
 pytest-cov
 pytest-mock
 tox
-twine
\ No newline at end of file
+twine
+pytest-helpers-namespace
```

Comment:
It's back! Also, you should move this requirement to be in alphabetical order.

Reply:
Fixed